components/Card/Card.tsx

```diff
@@ -55,6 +85,48 @@ export const Card = (CardProps: CardProps) => {
       {cardItems?.length && cardItems?.length > 3 && (
         <SeeMore onClick={handleSeeMoreClick} />
       )}
+      {cardObjectsToDisplay?.length && (
+        <CardBootstrap.Body className="px-0">
+          <CardBootstrap.Text className={styles.body}>
+            <Container>
+              {cardObjectsToDisplay.map((row, idx) => (
+                <Row key={idx}>
+                  {row.map((col, idxx) => {
+                    return (
+                      <Col key={idxx} xs={4} md={4} lg={4}>
+                        <div className="p-5 mx-5">
+                          <LabeledIcon
+                            key={idxx}
+                            idImage={`https://malegislature.gov/Legislators/Profile/170/${col.id}.jpg`}
+                            mainText="Sponsor"
+                            subText={
+                              <a
+                                href={`https://malegislature.gov/Legislators/Profile/${col.id}`}
+                              >
+                                {col.Name}
+                              </a>
+                            }
+                          />
+                        </div>
+                      </Col>
```

Comment:
This snippet goes into a SponsorRow component, and the cardObjectsToDisplay map is the `items` prop